src/main/constraints/105_default_bags.sql

```diff
@@ -0,0 +1,11 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE default_bags
+    ADD CONSTRAINT default_bags_user_id_fkey
+    FOREIGN KEY (user_id)
+    REFERENCES users(id) ON DELETE CASCADE;
+
+ALTER TABLE default_bags
+    ADD CONSTRAINT default_bags_bag_id_fkey
+    FOREIGN KEY (bag_id)
+    REFERENCES bags(id) ON DELETE CASCADE;
```

Comment:
A uniqueness constraint on the `user_id` and `bag_id` columns might be helpful here, just to ensure that we don't encounter a situation where a user has multiple default bags.

Reply:
Thanks for catching that. I added the uniqueness constraint in the tables/105_default_bags.sql file where the table is defined.